src/test_driver/tizen/testing_process.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
can you integrate with build_examples.py ?

if running something like `tizen-tests` it should allow me to run the tests.
Intent is to have a common interface for running things

Reply:
Hi @andy31415, thanks for your questions. You're right documentation should be written. I'm have question for you, could you tell me more about why integrations with build_example.py is necessary for you? I'm look on esp32 and there build_example.py build only new target with testing app, not running tests. They have script for running tests something like we have. In our Tizen tests we don't actually have new target testing app like esp32. We only add new docker container with Tizen Qemu image for running examples applications and testing them using "chip-tool" like on physical hardware. For running Qemu and simple tests we wrote python scripts.

Reply:
I am generally asking for consistency/convenience. Most developers working on the core SDK are not experts in one particular setup, so reading particular examples for every platform is not convenient. If we have a standard `run esp32-tests` and `run tizen-tests` and such then anyone can just use the vscode docker image and run things. Having custom scripts and platform-specific documentation on the other hand adds friction.

If you just update docker images, I believe this is ok and no integration is needed. If you add a `run_tests.sh` script, this means the tests are scriptable and I am asking for standardization.

Esp32 scripts were created before we tried to standardize compilation instructions, so some things are missing. New things however are encouraged to be uniform from the start.

Reply:
> New things however are encouraged to be uniform from the start.

Running tests with the "build_example.py" script is kinda nonintuitive, IMHO (even if there will be a "run" subcommand). Maybe we could add another script for running platform tests? Or maybe already existing `run_test_suite.py` could be adapted to run such tests? Right now Tizen tests are based on a simple script (commissioning + testing light app on/off cluster). In the long run, however, I'd like to run Matter test suite on Tizen (powered by QEMU).

Reply:
I understand we need a simpler way to run the tests. I am looking at the existing scripts in the directory ``. / scripts/tests`` and I think the existing scripts can be complicated to integrate with testing using Qemu. 
Do you think it's worth adding a new script to run Qemu-specific tests? Maybe like  ``run_test_qemu.py`` or  ``run_test_qemu.sh`` in directory ``. / scripts/tests``? In this case, any test running in Qemu can be performed using a schema and run from this script. Or maybe you have a better way?

Reply:
Resolving: can add this later as things are self-contained inside tizen for now.
I still believe that we should eventually add things into build_examples as an entry point for everything, so we do not need to remember various commands that are platform specific.